common.blocks/forum-form/forum-form.bemtree.js

```diff
@@ -0,0 +1,61 @@
+block('forum-form').content()(function() {
+    var i18n = this.require('i18n');
+
+    return [
+        {
+            elem: 'forum-create',
+            content: [
+                {
+                    elem: 'header',
+                    content: {
+                        block: 'radio-group',
+                        mods: { theme: 'islands', size: 'm', type: 'button' },
+                        name: 'radio-button',
+                        val: 1,
+                        options: [
+                            { 
+                                val: 1,
+                                icon: {
+                                    block: 'icon',
+                                    mods: { view: 'markdown', size: '24' }
+                                }
+                            },
+                            {
+                                val: 2,
+                                disabled: true,
+                                icon: {
+                                    block: 'icon',
+                                    mods: { view: 'preview', size: '24' }
+                                }
+                            }
+                        ]
+                    }
+                },
+                {
+                    elem: 'body',
+                    content: [
+                        {
+                            block: 'textarea',
+                            name: 'body',
+                            mods: { theme: 'islands', size: 'l', width: 'available' },
+                            mix: [{ block: 'forum-form', elem: 'view' }, { block: 'forum-form', elem: 'view_visible' }],
```

Comment:
how can I mix a block with element which has modifier(so, I wont to have like this: forum-form__view_visible)? I could not find this decision :(